qubesmanager/global_settings.py

```diff
@@ -36,6 +37,52 @@
 
 qmemman_config_path = '/etc/qubes/qmemman.conf'
 
+def _run_qrexec_repo(service, arg=''):
+    # Fake up a "qrexec call" to dom0 because dom0 can't qrexec to itself yet
+    cmd = '/etc/qubes-rpc/' + service
+    p = subprocess.run(
+        ['sudo', cmd, arg],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
+    )
+    assert not p.stderr
+    assert p.returncode == 0
```

Comment:
Ah yeah, I used `assert` because I assumed that that _would_ never happen, at least not if dom0 wasn't corrupted or misconfigured in some way... but I guess that qrexec policy could simply be set to denied by the administrator, right? Is that the impetus here?

Reply:
Yes, policy is one possibility. But some configuration issue (missing repository definition for example) could happen too.